extensions/wikia/AssetsManager/AssetsConfig.class.php

```diff
@@ -75,24 +75,12 @@ public static function getJQueryUrl( $combine, $minify, $params ) {
 	private function load() {
 		if ( empty( self::$mConfig ) ) {
 			wfProfileIn( __METHOD__ );
-			include( 'config.php' );
+			include( __DIR__ . '/config.php' );
+
 			/* @var $config Array */
 			self::$mConfig = $config;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$config" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)